### PR TITLE
Susan's greeting

### PR DESCRIPTION
**Pull request by NicMcPhee**
This adds a greeting for Susan.

### CODE REVIEW

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ public String generateOutput() {
 
     builder.append(kkSaysHello());
     builder.append(nicSaysHello());
+    builder.append(susanSaysHi());
```
Comment (day00090) [inline src/main/java/hellos/Hellos.java]: This is correctly alphabetized.

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -33,4 +34,8 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+  private String susanSaysHi() {
+    return "Susan says 'Welcome!'\n";
+  }
+
```
Comment (day00090) [inline src/main/java/hellos/Hellos.java:97]: Given that this matches the format of the other greetings and the tests passed, I think that you're ready to go!

**Review by morrisschooljm17 — DISMISSED** [review]
Looks good to me!

**Review by day00090 — DISMISSED** [review]

**Review by morrisschooljm17 — DISMISSED** [review]
Looks good

File: src/main/java/hellos/Hellos.java (hunk 1 above)
Comment (day00090) [reply src/main/java/hellos/Hellos.java]: The alphabetization remains correct.

File: src/main/java/hellos/Hellos.java (hunk 2 above)
Comment (day00090) [reply src/main/java/hellos/Hellos.java:97]: As before, the method returns a correctly formatted string, utilizing a name, "says", a greeting, and a tag for a new line.

**Review by day00090 — DISMISSED** [review]

**Review by dahlgrenthomas — DISMISSED** [review]
Looks good

**Review by morrisschooljm17 — DISMISSED** [review]
looks good

**Review by JacobJenness — APPROVED** [review]

**Pull request merged**